tests/lms/views/decorators/test_h_api.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import json
+import mock
+import pytest
+
+from pyramid.httpexceptions import HTTPBadGateway
+from pyramid.httpexceptions import HTTPBadRequest
+from pyramid.httpexceptions import HTTPGatewayTimeout
+import requests.exceptions
+
+from lms.views.decorators.h_api import create_h_user
+from lms.util import MissingToolConsumerIntanceGUIDError
+from lms.util import MissingUserIDError
+
+
+@pytest.mark.usefixtures("post", "util")
+class TestCreateHUser:
+    def test_it_400s_if_no_oauth_consumer_key_param(self, create_h_user, pyramid_request):
+        del pyramid_request.params["oauth_consumer_key"]
+
+        with pytest.raises(HTTPBadRequest, match="oauth_consumer_key"):
+            create_h_user(pyramid_request, mock.sentinel.jwt)
+
+    def test_it_continues_to_the_wrapped_func_if_feature_not_enabled(self, create_h_user, pyramid_request, wrapped):
```

Comment:
Would `returns` be a more descriptive term than `continues` here?

Reply:
Well, it doesn't _return_ the wrapped function (as in return the function itself), it calls the wrapped function with the right args and then returns what the wrapped function returned. Potential confusion there since we're dealing with decorators here, and plenty of functions themselves being returned is going on. I couldn't think of a better verb to cover calls the wrapped function with the right args and then returns what the wrapped function returned.